site/docs/master/code-standards.md

```diff
@@ -64,6 +64,14 @@ mockery -name=Restorer
 
 Might need to run `make update` to update the imports.
 
+## Kubernetes Labels
```

Comment:
not sure if calling out that this doesn't apply to annotations is necessary.
WDYT?

Reply:
I added a clarification.